### ISSUE TITLE
some methods are missing: IsPartitionTail(marker bool, payload []byte) bool

### ISSUE THREAD

**Issue by hktalent** [pion/rtp#271]
some methods are missing: IsPartitionTail(marker bool, payload []byte) bool

github.com/pion/rtp v1.8.6 // indirect

```
"github.com/pion/rtp/codecs"

go rtpToTrack(videoTrack, &codecs.VP8Packet{}, 90000, 5004)
rtpToTrack(audioTrack, &codecs.OpusPacket{}, 48000, 5006)
```
but 
https://github.com/pion/rtp/blob/bc5124c9d0d0b027de8bf84f1d50bb39619b1728/codecs/common.go#L17
have it

change code to ?:
```

type OpusPacket struct {
	codecs.OpusPacket
}

// IsPartitionHead
func (r *OpusPacket) IsPartitionHead(_ []byte) bool {
	return true
}

// IsPartitionTail
func (r *OpusPacket) IsPartitionTail(_ bool, _ []byte) bool {
	return true
}

type VP8Packet struct {
	codecs.VP8Packet
}

func (r *VP8Packet) IsPartitionTail(_ bool, _ []byte) bool {
	return true
}

go rtpToTrack(videoTrack, &VP8Packet{}, 90000, 5004)
rtpToTrack(audioTrack, &OpusPacket{}, 48000, 5006)

```

**Comment by at-wat** [comment]
Hi @hktalent

What is your problem?
As `audioDepacketizer` is embedded to `OpusPacket`, `OpusPacket` implements `IsPartitionHead/IsPartitionTail`.
It's correct that `IsPartitionHead/IsPartitionTail` always return true since Opus RTP packet doesn't allow fragmentation.

**Comment by hktalent** [comment]
@at-wat Thank you very much for your enthusiastic answer
When I used the following example, I found that the interface was not fully implemented.
https://github.com/pion/example-webrtc-applications/blob/f939bc71df2ae20b23049475fc8277deafe794aa/rtmp-to-webrtc/main.go#L92

Thank you very much
Let me know that it is correct to always return true


In addition, for this example, I tried to use mkfifo to create two pipe files to receive the output. I don’t know if there is any problem.
https://github.com/pion/example-webrtc-applications/tree/master/rtmp-to-webrtc
